File: Level 1/예산.js
function solution(d, budget) {
    let answer = 0;
    d.sort((a,b)=>a-b);
    let tmp = 0;
    for(let i=0;i<d.length;i++){
        if(tmp + d[i] > budget)break;
        tmp += d[i];
        answer++;
    }
    return answer;
}

File: Level 1/예산.cpp
#include <iostream>
#include <algorithm>
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> d, int budget) {
    int answer = 0;
    sort(d.begin(),d.end());
    int sum = 0;
    for(int i=0;i<d.size();i++){
        if(d[i] <= budget){
            budget -= d[i];
            answer+=1;
        }
        else break;
    }
    return answer;
}
